components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -949,6 +971,8 @@ def start_log_viewer_webui(
         "-u", f"{os.getuid()}:{os.getgid()}",
     ]
     # fmt: on
+    container_cmd.extend(container_cmd_extra_opts)
```

Comment:
Nit: I think we can drop the `# Start container` at the top of this block since it's a little misleading (the place we actually start the container is much further down and separated in a completely different block).